server/repository/mongodb/service_commitments.py

```diff
@@ -0,0 +1,48 @@
+"""
+Module for handling MongoDB operations related to service commitments.
+Provides methods to insert and fetch service commitments from the database.
+"""
+
+from config.mongodb_config import get_db
+
+
+class MongoRepoCommitments:
+    """
+    A MongoDB repository for managing service commitments.
+    """
+    def __init__(self):
+        """
+        Initialize the repository by setting up the database and collection.
+        """
+        self.db = get_db()
+        self.collection = self.db.service_commitments
+
+    def insert_service_commitments(self, commitments):
+        """
+        Inserts multiple service commitments into the database.
+
+        Args:
+            commitments (list): A list of dictionaries 
+            representing the service commitments.
+
+        Returns:
+            list: A list of inserted IDs for the commitments.
+        """
+        result = self.collection.insert_many(commitments)
```

Comment:
We want mongorepo to generate its own unique _id for each service commitment, instead of using our system generated uuid. To do that, you'll want to remove the "_id" field from the provided JSON structure before inserting into the database. See an example of how that's done in the repository/mongorepo.py file's add function.